src/phases/execution/AggregateExpressions.yml

```diff
@@ -0,0 +1,25 @@
+SchemaVersion: 2018-07-01
+Owner: "@mongodb/query"
+Description: |
+  This file defines a template to use in aggregation expression performance tests.
+
+ExpressionTemplate:
+  Name: {^Parameter: {Name: "name", Default: ""}}
+  Type: RunCommand
+  Database: &db {^Parameter: {Name: "database", Default: test}}
+  Threads: {^Parameter: {Name: "threads", Default: 1}}
+  Phases:
+    OnlyActiveInPhases:
+      Active: {^Parameter: {Name: "active", Default: [0]}}
+      NopInPhasesUpTo: {^Parameter: {Name: "nopInPhasesUpTo", Default: 63}}
+      PhaseConfig:
+        Repeat: {^Parameter: {Name: "repeat", Default: 10}}
+        Database: *db
+        Collection: &coll {^Parameter: {Name: "collection", Default: Collection0}}
+        Operations:
+        - OperationMetricsName: {^Parameter: {Name: "operationMetricsName", Default: OperationMetric}}
+          OperationName: RunCommand
+          OperationCommand:
```

Comment:
It has come to my attention that, at the moment, we should prefer running finds to aggregates in genny. This is because the [CrudActor will run a find, get a cursor, and exhaust it.](https://github.com/mongodb/genny/blob/5cae17c654f435cd082f94a63a1a586fe1bf01d6/src/cast_core/src/CrudActor.cpp#L642-L652) (RunCommand will only establish the cursor: https://github.com/mongodb/genny/blob/5cae17c654f435cd082f94a63a1a586fe1bf01d6/src/cast_core/src/RunCommand.cpp#L164-L187

this is all to say, this should be converted to use the CrudActor and specify the projection in the find language (something like so: https://github.com/mongodb/genny/blob/5cae17c654f435cd082f94a63a1a586fe1bf01d6/src/workloads/execution/ExpressiveQueries.yml#L172)

Reply:
Done.

Reply:
per offline discussion, this still doesn't work because the RunCommand actor won't exhaust the cursor. Though the CrudActor can do this for find commands, it does not currently support projections (see https://jira.mongodb.org/browse/EVG-17806). As such, we are electing to wait until https://jira.mongodb.org/browse/PERF-3485 is merged to use the CrudActor to run an aggregate command.

Reply:
Done. PERF-3485 is merged, using CRUD actor with aggregate command.

Reply:
Looks good! One suggestion: instead of specifying 

`[{$project: {out: <expression> }}]`

You could parameterize expression:

`Pipeline: [{$project: {output: {^Parameter: {Name: "Expression", Default: {}}}}]`

(Feel free to skip if this is too much of a refactor)